model-api-gen-gradle-test/typescript-generation/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
 
 tasks.named("npm_run_build") {
     dependsOn(":metamodel-export:generateMetaModelSources")
+    dependsOn("npm_run_test")
```

Comment:
To make the NPM install succeed, you need to add something like in here: https://github.com/modelix/modelix.core/blob/da591c859f031895661e78ee8e42f1d8b61aa424/model-api-gen-gradle-test/vue-integration/build.gradle.kts#L8

Reply:
Thanks for the help!